docs/changelog/89457.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 89457
+summary: Do not inject MigrateAction if it is not supported
+area: ILM+SLM
+type: enhancement
+issues: []
```

Comment:
I would remove the changelog because there is no impact to the user. Even thought the `MigrateAction` was injected in the beginning it was filtered out later in https://github.com/elastic/elasticsearch/blob/be7c7415627377a1b795400fb8dfcc6cbdf0e322/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/LifecyclePolicy.java#L239/

Reply:
ChangeLog removed